docs/source/app-dev/services.rst

```diff
@@ -59,6 +59,23 @@ Commands can be labeled with two application-specific IDs, both of which are ret
 
 For full details, see :ref:`the proto documentation for the service <com.digitalasset.ledger.api.v1.CommandSubmissionService>`.
 
+.. _command-submission-service-deduplication:
+
+Command deduplication
+---------------------
+
+The command submission service deduplicates submitted commands:
+
+- Applications can assign a "time to live" (TTL) value to each command.
+- Duplicate command submissions (using the same submitter and command ID) will be ignored until either the TTL of the original command has elapsed or the original submission was rejected, whichever comes first.
```

Comment:
application id is currently also used, so this should be the triple `(submitter, application-id, command-id)`, unless we're changing that with the new time model?

Reply:
The conclusion from the design document was that the application ID should not be part of the deduplication key.

Reply:
I stand corrected. We'll need to fix this in kvutils. Can you drive that change @rautenrieth-da? Should be pretty straightforward since we can still break backwards compatibility.